phys2bids/cli/run.py

```diff
@@ -22,136 +22,186 @@ def _get_parser():
-    optional.add_argument('-thr', '--threshold',
-                          dest='thr',
-                          type=float,
-                          help='Threshold to use for trigger detection. '
-                               'If "ntp" and "TR" are specified, phys2bids '
-                               'automatically computes a threshold to detect '
-                               'the triggers. Use this parameter to set it manually. '
-                               'This parameter is necessary for multi-run recordings. ',
-                          default=None)
-    optional.add_argument('-pad', '--padding',
-                          dest='pad',
-                          type=float,
-                          help='Padding in seconds used around a single run '
-                               'when separating multi-run session files. '
-                               'Default is 9 seconds.',
-                          default=9)
-    optional.add_argument('-chnames', '--channel-names',
-                          dest='ch_name',
-                          nargs='*',
-                          type=str,
-                          help='Column header (for json file output).',
-                          default=[])
-    optional.add_argument('-yml', '--participant-yml',
-                          dest='yml',
-                          type=str,
-                          help='full path to file with info needed to generate '
-                               'participant.tsv file ',
-                          default='')
-    optional.add_argument('-debug', '--debug',
-                          dest='debug',
-                          action='store_true',
-                          help='Only print debugging info to log file. Default is False.',
-                          default=False)
-    optional.add_argument('-quiet', '--quiet',
-                          dest='quiet',
-                          action='store_true',
-                          help='Only print warnings to log file. Default is False.',
-                          default=False)
-    optional.add_argument('-v', '--version', action='version',
-                          version=('%(prog)s ' + __version__))
+    required = parser.add_argument_group("Required Argument:")
+    required.add_argument(
+        "-in",
+        "--input-file",
+        dest="filename",
+        type=str,
+        help="The name of the file containing physiological "
+        "data, with or without extension.",
+        required=True,
+    )
+    optional.add_argument(
+        "-info",
+        "--info",
+        dest="info",
+        action="store_true",
+        help="Only output info about the file, don't process. "
+        "Default is to process.",
+        default=False,
+    )
+    optional.add_argument(
+        "-indir",
+        "--input-dir",
+        dest="indir",
+        type=str,
+        help="Folder containing input. " "Default is current folder.",
```

Comment:
Like here

Reply:
Yes, this is the one error that `black` makes, in my experience.